src/main/java/com/sonsminpark/auratalkback/domain/user/entity/User.java

```diff
@@ -0,0 +1,66 @@
+package com.sonsminpark.auratalkback.domain.user.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+@Builder
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false, unique = true, length = 100)
+    private String email;
+
+    @Column(nullable = false, length = 100)
+    private String password;
+
+    @Column(nullable = false, length = 15)
+    private String username;
+
+    @Column(nullable = false, length = 15)
+    private String nickname;
+
+    @ElementCollection
+    @CollectionTable(name = "user_interests", joinColumns = @JoinColumn(name = "user_id"))
+    @Column(name = "interest")
+    private List<String> interests = new ArrayList<>();
+
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private UserStatus status;
+
+    @CreationTimestamp
+    private LocalDateTime createdAt;
+
+    @Column(nullable = false)
+    private boolean isDeleted;
+
+    @CreationTimestamp
+    private LocalDateTime deletedAt;
+
+    public void updateStatus(UserStatus status) {
+        this.status = status;
+    }
+
+    public void delete() {
+        this.isDeleted = true;
+    }
```

Comment:
```suggestion

    private LocalDateTime deletedAt;

    public void updateStatus(UserStatus status) {
        this.status = status;
    }

    public void delete() {
        this.isDeleted = true;
        this.deletedAt = LocalDateTime.now();
    }
```
   
@CreationTimeStamp 대신 이렇게 수정하는건 어떤가요?

Reply:
deletedAt 필드를 @CreationTimestamp로 설정해서 엔티티 생성 시점에 자동으로 값이 설정되게 했었는데, 생각해보니까 말씀해주신 것처럼 deletedAt을 사용자가 삭제될 때 설정되게 하는 게 좋은 것 같아요.